shadow-dom/reference-target/tentative/popovertarget.html

```diff
@@ -11,29 +11,42 @@
 </head>
 
 <body>
-  <button id="toggle-button" popovertarget="x-popover-1">Toggle the popover</button>
+  <button id="toggle-button-1" popovertarget="x-popover-1">Toggle the popover</button>
   <x-popover-1 id="x-popover-1">
     <template shadowrootmode="open" shadowrootreferencetarget="popover-1">
       <div id="popover-1" popover>Popover content inside shadow root</div>
     </template>
   </x-popover-1>
 
+  <button id="toggle-button-2" popovertarget="x-popover-2">Toggle the popover</button>
+  <x-popover-2 id="x-popover-2"></x-popover-2>
+
+  <script>
+    const customPopover2 = document.querySelector('x-popover-2');
+    customPopover2.attachShadow({ mode: 'open', referenceTarget: 'popover-2' });
```

Comment:
```suggestion
    customPopover2.attachShadow({ mode: 'open', referenceTarget: 'popover-1' });
```

Id here needs to match what's in the shadow root.

This ID is also hardcoded in `testPopoverTarget` on line 34, so maybe the ID in the shadow DOMs could just change to something like "target" for both tests?